main/src/main/res/values/strings_not_translatable.xml

```diff
@@ -73,21 +73,21 @@
     <!-- recent contributors, meaning: last 12 months (app.) -->
     <!-- one line per contributor, format is: <name>|<link>|<contribution identifiers>| -->
     <!-- (contribution identifiers are: c=code, d=documentation, g=graphics, i=infrastructure & CI, p=project leader, s=support, t=testing) -->
-    <!-- currently listed: contributions 2022-10 to 2023-09 -->
+    <!-- currently listed: contributions 2023-01 to 2023-12 -->
     <string translatable="false" name="contributors_recent">
     baiti|https://github.com/baiti|c|
     bekuno|https://github.com/bekuno|ci|
     eddiemuc|https://github.com/eddiemuc|cd|
     fm-sys|https://github.com/fm-sys|cg|
     kumy|https://github.com/kumy|c|
     Lineflyer|https://github.com/Lineflyer|ptsdi|
-    liroP|https://github.com/liroP|c|
     moving-bits|https://github.com/moving-bits|cdgs|
     mucek4||cip|
     murggel|https://github.com/murggel|cdg|
-    mzylowski|https://github.com/mzylowski|c|
     NotWoods|https://github.com/NotWoods|c|
-    osmpaws|https://github.com/osmpaws|c|
+    mortenbruhn|https://github.com/mortenbruhn|c|
+    mzylowski|https://github.com/mzylowski|c|
```

Comment:
This should be sorted alphabetically, right?

Reply:
argh! should have been... I'll fix it. Thanks for notifying me.